Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.1
+FROM ruby:3.3.4
```

Comment:
I'm not sure where this Docker image is used. Do you know?

Reply:
Used for local dev/build environments is all I could determine.  The AI even used it to serve it up to me.